packages/server/src/main/resources/messages.properties

```diff
@@ -0,0 +1,21 @@
+generic.unknown = Oops! Something went wrong. Please try again later.
+
+asset.not-found = We were unable to find the requested asset. Please ensure that the ID you supplied is correct or contact support.
+
+code.not-found = We were unable to find the requested code. Please ensure that the code ID you supplied is correct or contact support.
+code.unable-to-update = We were unable to update the code asset. Please try again later.
+code.unable-to-delete = We were unable to delete the code asset. Please try again later.
+
+postgres.service-unavailable = Terarium's data storage is temporarily unavailable. Please try again later.
```

Comment:
should the apostrophe be escaped: `\'`?

Reply:
I think this is just github's problem... not the application. I don't get any errors from intellij